client/src/includes/usebar.test.ts

```diff
@@ -0,0 +1,117 @@
+import { fireEvent } from '@testing-library/dom';
+
+jest.useFakeTimers();
+
+describe('handleTriggerKeyDown', () => {
+  let trigger: HTMLElement;
+  let showUserbar: jest.Mock;
+  let setFocusToFirstItem: jest.Mock;
+  let setFocusToLastItem: jest.Mock;
+
+  beforeEach(() => {
+    // Mocks
+    showUserbar = jest.fn();
+    setFocusToFirstItem = jest.fn();
+    setFocusToLastItem = jest.fn();
+
+    // Criação do elemento "trigger"
+    trigger = document.createElement('button');
+    trigger.setAttribute('aria-expanded', 'false');
+    document.body.appendChild(trigger);
+
+    // Mock do `document.activeElement`
+    jest.spyOn(document, 'activeElement', 'get').mockImplementation(() => trigger);
+  });
+
+  afterEach(() => {
+    document.body.innerHTML = '';
+    jest.restoreAllMocks();
+    jest.clearAllTimers();
+  });
+
+  const handleTriggerKeyDown = (event: KeyboardEvent) => {
+    if (
+      trigger === document.activeElement &&
+      trigger.getAttribute('aria-expanded') === 'false'
+    ) {
+      switch (event.key) {
+        case 'ArrowUp':
+          event.preventDefault();
+          showUserbar(false);
+          setTimeout(() => setFocusToLastItem(), 300);
+          break;
+        case 'ArrowDown':
+          event.preventDefault();
+          showUserbar(false);
+          setTimeout(() => setFocusToFirstItem(), 300);
+          break;
+        default:
+          break;
+      }
+    }
+  };
```

Comment:
The whole test file seems to be testing this function implementation that you wrote inside the test file, which doesn't seem to make much sense.